### PR TITLE
Add support for quote_mint_redeem_softcap

### PR DESCRIPTION
**Pull request by acamill**
Support for the new IX

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -604,6 +604,25 @@ export class UXDClient {
...
   }
 
+  public createSetMangoDepositoryQuoteMintAndRedeemSoftCapInstruction(
+    softCap: number,
+    controller: Controller,
+    depository: MangoDepository,
+    authority: PublicKey,
+    options: ConfirmOptions
+  ): TransactionInstruction {
+    const softCapNativeBN = new BN(
```
Comment (Orelsanpls) [inline src/client.ts]: You don't need to wrap the BN

`const softCapNativeBN = new BN(softCap).mul(new BN(10).pow(new BN(depository.quoteMintDecimals)));`

File: src/client.ts [hunk 2]
```diff
@@ -604,6 +604,25 @@ export class UXDClient {
...
+      new BN(softCap).mul(new BN(10).pow(new BN(depository.quoteMintDecimals)))
+    );
+    return this.instruction.setMangoDepositoryQuoteMintAndRedeemSoftCap(softCapNativeBN, {
+      accounts: {
```
Comment (Orelsanpls) [inline src/client.ts]: When the key have the same name as the variable, you can do:

```
     accounts: {
        authority,
        controller: controller.pda,
      },
      options,
```

this is called `shorthand`

Comment (acamill) [reply src/client.ts]: Fixed! (also have this in Rust no excuse hehe)

**Comment by cnek** [comment]
due to merge order, idl changes has already reflected on https://github.com/UXDProtocol/uxd-client/pull/13

**Pull request merged**